### ISSUE TITLE
Can´t connect devices with the same subnet created by docker in installation

### ISSUE THREAD

**Issue by jonasrita99** [Ubiquiti-App/UNMS#135]
I´m having a issue with adding devices that have the same ip address/subnet from the routes that Docker creates during the installation. (e.g: I want to add a device with the address 172.17.1.1, but it doesn´t work because the server already created the route "172.17.0.0/16" during the installation). These subnets are overlaping and i can only add devices that have differents management subnets. 

I´ve tried to remove these routes, but when i do that, the web server doesn´t work anymore.

**Comment by radekskrivan** [comment]
@jonasrita99  Hello. I believe the only option now is to reinstall your UNMS with an [optional tag](https://help.ubnt.com/hc/en-us/articles/360000119728-UNMS-Optional-Installation-Steps#8) which will change UNMS IP address.

**Comment by prezeskk** [comment]
It don't work
You must change options in /etc/default/docker.
Add for example:
```
DOCKER_OPTS="--bip=172.18.1.1/16 --fixed-cidr=172.18.1.1/16"
```
and restart docker

**Comment by radekskrivan** [comment]
@jonasrita99 Hello, how is this issue developing. Did you manage to change the docker IP so that it doesn't collide with your devices?

**Comment by radekskrivan** [comment]
@jonasrita99  It seems that most systems do not use `/etc/default/docker`
Please try to add 
```
{
  "bip": "10.18.0.1/16",
  "fixed-cidr": "10.18.0.1/16"
}
```
to the` /etc/docker/daemon.json`
After that restart docker `systemctl restart docker`
